### ISSUE TITLE
Checkbox and radio button can't read as a whole with the label by NVDA in browse mode

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#5068]
## Describe the accessibility issue

While using the arrow key to navigate the Clarity checkboxes or radio buttons, the component is not read as a whole.

For a checkbox, it read `checkbox, not checked.` and the label requires an extra arrow keystroke.

The same for radio buttons.

## Expected behavior

Please refer to the attached video, expect to read the component as a whole as the native checkbox element.
[label not read in browse mode.mp4.zip](https://github.com/vmware/clarity/files/5259367/label.not.read.in.browse.mode.mp4.zip)

## Versions

**App**

- Angular: [8]
- Node: [10]
- Clarity: [2]

**Device:**

- OS: [Windows 10]
- Browser [firefox]
- Version [80]
- Screen Reader: [NVDA]

**Comment by hippee-lee** [comment]
@chlane - can you weigh in on the expected behavior in this component?

**Comment by chlane** [comment]
This is not a bug. Our current implementation below is accessible, the label is read when the checkbox gains focus. 

`
<input type="checkbox" clrcheckbox="" name="test3" required="" value="option1" id="clr-form-control-3" class="ng-untouched ng-pristine ng-invalid" aria-describedby="clr-form-control-3-helper"><label class="clr-control-label" for="clr-form-control-3">Option 1</label>`

Additionally, it passes the Wave and Andi accessibility checkers. @KingMario if you think NVDA should behave differently I suggest reporting it at https://github.com/nvaccess/nvda/issues.

**Comment by hippee-lee** [comment]
Thanks @clane. Closing as invalid.
